src/data/real/ereal.lean

```diff
@@ -514,3 +526,36 @@ lemma to_real_mul : ∀ {x y : ereal}, to_real (x * y) = to_real x * to_real y
 | ⊥ ⊥ := by simp
 
 end ereal
+namespace tactic
+
+open positivity
+
+meta instance : has_to_string strictness :=
+⟨λ s, match s with
+  | positive p := "strictness.positive (" ++ to_string p ++ ")"
+  | nonnegative p := "strictness.nonnegative (" ++ to_string p ++ ")"
+  end⟩
+
+meta instance : has_to_format strictness := ⟨λ s, to_string s⟩
```

Comment:
This is apparently unused, and in any case I guess it doesn't belong here.  Is it leftover from something for testing purposes?

Reply:
Indeed, I was debugging using this. I will remove it from this PR, but would you like having it in another PR?

Reply:
Now included as part of #16529